sources/3.0.0/sections/11-data_product_delivery.adoc

```diff
@@ -46,6 +46,32 @@ Three types of dataset files may be produced and contained within an exchange se
 * New edition of a dataset: Includes new information. New editions must cover at least the same area as its predecessor.
 * [[canc]]Cancellation: The dataset is cancelled and no longer available to be displayed or used.
 
+[[subsec-metadata-for-cancellation]]
+===== Metadata for cancellation
+S-102 uses the fileless cancellation method described in <<iho-s100,part=17,clause=17-4.4.1>>. For a cancellation, set:
+
+* _fileName_ = _fileName_ of the cancelled dataset
+
+* _purpose_ = _cancellation_
+
+* edition number = 0
```

Comment:
I don't like the idea of using edition numbers for the S-102. They are not necessary and only make things more complicated. In addition, the edition number is also missing in the HDF5 file itself.
See Issue #89

Reply:
Agreed and modified in the code